torchserve/tests/fixtures/management/events/models_without_2_2.json

```diff
@@ -0,0 +1,20 @@
+{
+  "models": [
+    {
+      "modelName": "linear_regression_1_1",
+      "modelUrl": "linear_regression_1_1.mar"
+    },
+    {
+      "modelName": "linear_regression_1_2",
+      "modelUrl": "linear_regression_1_2_v3.mar"
+    },
+    {
+      "modelName": "linear_regression_2_2",
```

Comment:
File name says without 2.2 but there is 2.2 present

Reply:
Good catch! It's `models_without_3_2.json`